axelrod/tournament.py

```diff
@@ -234,20 +244,18 @@ def _write_interactions_to_dict(self, results):
                     self.interactions_dict[index_pair] = [interaction]
                 self.num_interactions += 1
 
-    def _run_parallel(self, processes: int=2, progress_bar: bool = False
-                      ) -> bool:
+    def _run_parallel(self, processes: int=2) -> bool:
         """
         Run all matches in parallel
 
         Parameters
         ----------
 
-        progress_bar : bool
-            Whether or not to update the tournament progress bar
+        processes : int
+            How many processes to use.
         """
         # At first sight, it might seem simpler to use the multiprocessing Pool
```

Comment:
i keep forgetting to ask.  should this comment now be removed?  when i first started working on this, i investigated Pool because of the original comment.  It pointed to the possibility of using Pool if python changed.  As the original code is simply more efficient, there may be no need to mention Pool at all.  My updated comment may simply be useless noise.  (who, after all, would not use a conventional tool if it worked better)